python/pyspark/sql/context.py

```diff
@@ -95,7 +119,7 @@ def __init__(self, sparkContext, sparkSession=None, jsqlContext=None):
             SQLContext._instantiatedContext = self
 
     @property
-    def _ssql_ctx(self):
+    def _ssql_ctx(self):  # type: ignore[no-untyped-def]
```

Comment:
If the purpose is validation of our own codebase, we should avoid `type: ignoring` internal methods.